### DISCUSSION TITLE
How to setup/enable iOS Notification Images in the expo project?

### DISCUSSION THREAD

**Discussion by takosunil** [invertase/react-native-firebase#7865] [Q&A]
We have the [documentation](https://rnfirebase.io/messaging/ios-notification-images) to display an image in an incoming notification for bare React Native. But I couldn't find any guidelines or documentation for the Expo.

I can follow the bare React Native guide for the Expo. But the problem is the iOS folder is being re-generated with every prebuild command and those files are not saved in source code.

Are there any steps or documentation on how we can display an image in an incoming notification for the Expo app?

**Comment by erdincakdogan** [comment]
Did you solve the problem ?
